### ISSUE TITLE
Title and scope of "Reactants" section

### ISSUE THREAD

**Issue by PeterHalling** [StephanM87/Strenda-biocatalysis#36]
I think these may not be quite right (just realised this now, was focussed on the contents of the section). We state that in this section “Each additive of a reaction must be defined like substrate, cosubstrate, cofator, etc. Exlcuded are only biocatalyst and the solvents". But the section title may be confusing, because the category surely includes salts, buffers, metal ions etc. None of these would normally be called “reactants”, as they are not changed in the enzymatic reaction. Probably a better section title would be "Ingredients" or "Components", perhaps with "of the Reaction Mixture" after it. "Components" has a rather precise definition in thermodynamic treatments, but I don't think we would be too outside this definition by using it here.

Then perhaps we need to think again about our reasons to treat certain components of the reaction mixture differently?  Certainly the biocatalyst(s) normally require description rather different than that for small molecules. So perhaps a more logical separation is between small molecules and macromolecules or other less well defined materials. I think excluding the solvent from the general Components section will lead to confusion. For example molten fats or vinyl acetate might be the largest fraction of the reaction mixture, with other components dissolved in them. So they should  be considered solvents, but are also reactants converted by the enzyme.

**Comment by DomMSNR** [comment]
Thanks for the comment. 
I agree with you about the name of the category. I also understand this category to list all the components that are involved or important in the reaction. Among them also salts and ions. I would not necessarily call these reactants. I consider a renaming of the category in "Components" would be therefore plausible.
To integrate the solvents here is a bit difficult. The solvents rather the description of the individual phases is currently part of the reaction conditions. There is a distinction between well mixed and flow/continuous. There, it makes sense to go into more detail about the individual phases, including the liquid phase/solvents.
If the solvent is part of the reaction, there is no reason not to list it under the category "components", is there?
Accordingly, only in the description would the word "solvents" be replaced, as it may be misleading here. --> "Each additive of a reaction must be defined like substrate, cosubstrate, coenzyme, salts, ions, etc. The biocatalyst and the detailed description of the phases are not listed under the reaction components."

**Comment by PeterHalling** [comment]
Thanks, that now looks OK to me - except I don't like the use of the word phase in this way, see separate issue.

**Comment by DomMSNR** [comment]
I have found the issue regarding the term "phase". It's good that you brought this up again. I'm sure another designation would be more appropriate.

**Comment by jmrohwer** [comment]
I agree that the "Components" section is more appropriate. However, I think there is scope for separate "Reactants" and "Products" classes that specifically refer to the components that are converted by the biocatalyst. These are different from normal assay components and play a special role, so I think they should have a special designation. It is also like that in "normal" STRENDA and as well as EnzymeML.

Whether "Reactant" and "Product" be instantiated as sub-class of "Component" or as top-level can be discussed. 

In addition they should be linked to the **Reaction** (see separate issue).

**Comment by PeterHalling** [comment]
Yes, Johann in right we should have attributes to identify reactants and products. But I think these fit better in the Reaction category he proposes, rather than in Components/Ingredients. I fully support the idea of a Reaction category, see comment on that Issue. I see Components/Ingredients as a statement of fact, what I put in the reaction mixture. The nature of the reaction is to some extent a matter of prediction and/or experimental results. Exactly what happens will be decided by the biocatalyst (and other chemistry), rather than me - there are plenty of examples where unexpected reactions happen as well as (or instead of) what was planned. So that's why I would keep it in a separate category. Components/Ingredients is the metadata that allows someone else to reproduce the experiment (perhaps to show that the reaction is in fact different from what was previously proposed!).
